crates/fullnode/src/runner.rs

```diff
@@ -569,7 +570,13 @@ where
                 .unwrap();
 
             let (sequencer_commitments, zk_proofs) =
-                self.extract_relevant_l1_data(l1_block.clone());
+                match self.extract_relevant_l1_data(l1_block.clone()).await {
+                    Ok(r) => r,
+                    Err(e) => {
+                        error!("Could not process L1 block: {}...skipping", e);
```

Comment:
Is it okay?

Reply:
if we try to parse every possibly-relevant transaction we ignore the ones that fail

we only return error on things like bitcoin da cannot get response for rpc request etc. yes this is ok. we don't do anything and retry later.

Reply:
If any tx fails in a block - we fail the block completely.